Library/Library.csproj

```diff
@@ -30,22 +30,24 @@
   <ItemGroup Condition="'$(OS)' == 'Windows_NT' OR '$(TargetFrameworkIdentifier)' == '.NETFramework'">
     <Compile Remove="Copied/Unix/*.cs" />
     <Compile Remove="**/*.Unix.cs" />
-    <Compile Remove="SafeProcessHandle.Linux.cs" />
+    <Compile Remove="SafeChildProcessHandle.Linux.cs" />
+    <Compile Remove="SafeChildProcessHandle.Unix.cs" />
```

Comment:
the check performed above (`<Compile Remove="**/*.Unix.cs" />`) will include this file

```suggestion
```